partner_brand/tests/test_partner_brand.py

```diff
@@ -0,0 +1,60 @@
+# © 2023 David BEAL @ Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import base64
+import os
+
```

Comment:
Can you please include the tag tests as in:
https://github.com/OCA/brand/pull/268/files